components/atoms/TopContributorCard/top-contributor-card.tsx

```diff
@@ -57,7 +57,7 @@ const TopContributorCard = ({ login, refreshLoggedInUser }: TopContributorCardPr
           onClick={() =>
             sessionToken
               ? handleFollowContributor()
-              : signIn({ provider: "github", options: { redirectTo: `${host}/${currentPath}` } })
+              : signIn({ provider: "github", options: { redirectTo: `${host}${currentPath}` } })
```

Comment:
refactor (non-blocking): You can use a new URL to avoid slash issues.

```suggestion
              : signIn({ provider: "github", options: { redirectTo: `${new URL(currentPath, host)}` } })
```